Command/DumpCommand.php

```diff
@@ -0,0 +1,190 @@
+                null
+            )
+            ->addOption(
+                'force',
+                null,
+                InputOption::VALUE_NONE,
+                'Suppresses prompt asking whether to continue.'
+            );
+    }
+
+
+    /**
+     * {@inheritdoc}
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     */
+    protected function execute (InputInterface $input, OutputInterface $output)
+    {
+        $dumper       = $this->getContainer()->get('becklyn_mysql_dump.services.dump');
+        $dumperConfig = $this->getContainer()->get('becklyn_mysql_dump.services.configuration');
+
+        $connections = $dumperConfig->getDatabases($input->getOption('connections'));
+        $backupPath  = $dumperConfig->getBackupDirectory($input->getOption('path'));
+
+        /** @var FormatterHelper $formatter */
+        $formatter = $this->getHelper('formatter');
+
+        // Print headline
+        $headline = $formatter->formatBlock(array('', '  Becklyn MySQL Database Dumper', ''), 'comment');
+        $output->writeln($headline);
+
+        // If there are no database connection information available we can't dump anything
+        if (empty($connections))
+        {
+            $error = $formatter->formatBlock(array('', '  No connection data found.  ', ''), 'error');
+            $output->writeln("\n{$error}\n");
+
+            return 1;
+        }
+
+        $unresolvedConnections = false;
+        $dbBackupPaths         = [];
+        $tableRows             = [];
+
+        /** @var Connection $connectionData */
+        foreach ($connections as $connectionName => $connectionData)
+        {
+            // Check if the connection could not be resolved.
+            if ($connectionData === null)
+            {
+                $unresolvedConnections = true;
+
+                $tableRows[] = [
+                    '<error>- unresolved -</error>', $connectionName, '-'
+                ];
+
+                continue;
+            }
+
+            // Preserve the designated file names for the actual backup process as an actual dump
+            // may take very long the actual file names would differ from the one we printed to the user
+            $dbBackupPaths[$connectionName] = $this->getBackupFilePath($backupPath, $connectionName, $connectionData);
+
+            $tableRows[] = [
+                $connectionData->getDatabase(),
+                $connectionName,
+                $dbBackupPaths[$connectionName]
+            ];
+        }
+
+        // Print a nice table with the database name and the actual target file path
+        $this->getHelper('table')
+             ->setHeaders(array('Database', 'Connection', 'Backup file'))
+             ->setRows($tableRows)
+             ->render($output);
+
+        // Check if any connections could not be resolved and then print an error to abort.
+        if ($unresolvedConnections)
```

Comment:
If you extract the table into its method, you won't have this info available.

Just use `in_array(null, $connections, true)` now instead.